apps/spruce/src/pages/version/taskDuration/TaskDurationTable.tsx

```diff
@@ -24,6 +26,8 @@ import { TaskDurationCell } from "./TaskDurationCell";
 const { getDefaultOptions: getDefaultFiltering } = ColumnFiltering;
 const { getDefaultOptions: getDefaultSorting } = RowSorting;
 
+type TaskDurationColumnData =
+  VersionTaskDurationsQuery["version"]["tasks"]["data"][0];
```

Comment:
let's use the `Unpacked` util
```suggestion
type TaskDurationColumnData =
  Unpacked<VersionTaskDurationsQuery["version"]["tasks"]["data"]>;
```